### PR TITLE
COPYRIGHT: remove hoedown license

### PR DESCRIPTION
**Pull request by nodakai**
Hoedown was removed in b96fef8411f

Also cleanup src/tools/tidy/src/main.rs

### CODE REVIEW

**Comment by steveklabnik** [comment]
Removing the hoedown stuff makes sense to me. @alexcrichton I am 99% sure these tidy changes do to, but wdyt?

File: src/tools/tidy/src/main.rs [hunk 1]
```diff
@@ -82,12 +82,9 @@ fn filter_dirs(path: &Path) -> bool {
         "src/llvm",
         "src/libbacktrace",
         "src/compiler-rt",
-        "src/rt/hoedown",
         "src/rustllvm",
         "src/rust-installer",
         "src/liblibc",
-        "src/tools/cargo",
-        "src/vendor",
```
Comment (TimNN) [inline src/tools/tidy/src/main.rs]: The removal of this line causes tidy errors during `dist-check`.

Comment (nodakai) [inline src/tools/tidy/src/main.rs]: In fact several issues are preventing me from completing `x.py test distcheck` locally... Just blindly reverted the last line

**Pull request merged**